modules/tinymce/src/core/main/ts/content/EditorContent.ts

```diff
@@ -5,14 +5,54 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Cell } from '@ephox/katamari';
+
+import { addDefaultGetFormats } from 'tinymce/core/content/GetContentImpl';
+import { addDefaultSetFormats } from 'tinymce/core/content/SetContentImpl';
+
+import Editor from '../api/Editor';
+import * as AddFormat from './AddFormat';
 import { Content, GetContentArgs, SetContentArgs } from './ContentTypes';
 import { getContent } from './GetContent';
 import { setContent } from './SetContent';
 
+interface EditorContent {
+  addFormat: (format: string, formatGetter: GetContentFormatter, formatSetter: SetContentFormatter) => void;
+  get: (args: Partial<GetContentArgs>) => Content;
+  set: (content: Content, args?: Partial<SetContentArgs>) => Content;
```

Comment:
Actually ignore this sorry, as I forgot these are on the public API. Since this logic is now public, it should also be moved to the `api/content/` directory instead. 

- [ ] Move file to `api` directory